pkg/streamingpromql/operators/functions/functions.proto

```diff
@@ -67,6 +67,8 @@ enum Function {
   FUNCTION_SIN = 58;
   FUNCTION_SINH = 59;
   FUNCTION_SORT = 60;
+  FUNCTION_SORT_BY_LABEL = 76;
```

Comment:
@charleskorn I elected to keep these in sorted order but picked the next unused ordinals. LMK if this isn't the right thing to do.